src/search/search.component.ts

```diff
@@ -125,9 +126,18 @@ export class Search implements ControlValueAccessor {
 	 */
 	@Input() clearButtonTitle = this.i18n.get().SEARCH.CLEAR_BUTTON;
 	/**
+	 * @deprecated in the next major carbon-components-angular version in favour of `valueChange`
 	 * Emits event notifying other classes when a change in state occurs in the input.
 	 */
 	@Output() change = new EventEmitter<SearchChange>();
+	/**
+	 * Emits an event when value is changed.
+	 */
+	@Output() valueChange = new EventEmitter<any>();
```

Comment:
In this case we know what types we're dealing with ...

```suggestion
	@Output() valueChange = new EventEmitter<string>();
```